web/models.py

```
@@ -2804,3 +2806,417 @@ class ScheduledPost(models.Model):
+            )
+
+            # Send email notification
+            try:
+                send_mail(
+                    notification_title,
+                    notification_message,
+                    settings.DEFAULT_FROM_EMAIL,
+                    [self.user.email],
+                    fail_silently=True,
+                )
+            except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)